backend/models/user.js

```diff
@@ -538,11 +539,13 @@ User.verify = async function (username, token, options) {
 	}
 
 	try {
-		const { customData: {firstName, lastName, email, billing, mailListOptOut} } = user;
+		const { customData: {firstName, lastName, email, billing, mailListOptOut, jobTitle, phoneNumber,
```

Comment:
we dont' want to store it in the database, just intercom. so you shouldn't need to change this line

Reply:
I think intercom entry is created after email verification and we make it based on what we have stored in the db after sign up.

Reply:
ah... stupid system.

@mosment3DRepo fyi you have to store it in the database.

In this case, @ChristosTsiotsias  can we group it in an embedded json i the database so it's less clunky

like `customData.extras` or something.

Reply:
yeap sure